Purchases/Public/IntroEligibility.swift

```diff
@@ -83,8 +83,10 @@ private extension IntroEligibilityStatus {
             return "Eligible for trial or introductory price."
         case .ineligible:
             return "Not eligible for trial or introductory price."
-        default:
-            return "Status indeterminate."
+
+        case .unknown: fallthrough
```

Comment:
TIL `fallthrough`

Reply:
No fallthrough by default is one of those small things that makes Swift a much better language.